<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineDialog</class>
 <widget class="QWidget" name="OnlineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>     人像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>未登录</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>701</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_login">
      <property name="text">
       <string>登录/注册</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_rmptUrl">
      <property name="text">
       <string>rmpt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_rmtpPlay">
      <property name="text">
       <string>播放rmpt流</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>701</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>40</number>
    </property>
    <item>
     <widget class="QPushButton" name="pb_tv">
      <property name="text">
       <string>地方台</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_recommendTV">
      <property name="text">
       <string>推荐影视</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_fresh">
      <property name="text">
       <string>刷新影视</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_upload">
      <property name="text">
       <string>上传视频</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_uploaded">
      <property name="text">
       <string>已上传视频</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>731</width>
     <height>391</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pv_tv1">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv2">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv3">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv4">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv5">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pv_tv6">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv7">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv8">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv9">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv10">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pv_tv11">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv12">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv13">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv14">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pv_tv15">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>111</height>
           </size>
          </property>
          <property name="text">
           <string>TV图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QMyMoveLable" name="ml_1" native="true">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>23</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_2" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>23</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_3" native="true">
     <property name="geometry">
      <rect>
       <x>297</x>
       <y>23</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_4" native="true">
     <property name="geometry">
      <rect>
       <x>434</x>
       <y>23</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_5" native="true">
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>23</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_8" native="true">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>142</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_9" native="true">
     <property name="geometry">
      <rect>
       <x>433</x>
       <y>142</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_6" native="true">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>142</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_7" native="true">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>142</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_10" native="true">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>142</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_13" native="true">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>262</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_14" native="true">
     <property name="geometry">
      <rect>
       <x>433</x>
       <y>262</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_15" native="true">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>262</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_11" native="true">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>262</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QMyMoveLable" name="ml_12" native="true">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>262</y>
       <width>130</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_change_name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>改名字</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMyMoveLable</class>
   <extends>QWidget</extends>
   <header location="global">qmymovelable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
